pkg/project/project.go

```diff
@@ -1,7 +1,7 @@
 package project
 
 var (
-	bundleVersion        = "7.0.1-dev"
+	bundleVersion        = "10.9.9"
```

Comment:
@xh3b4sd re our yesterday chat. Also versions like `0.0.0-xh3b4sd` can be used. I think that would be a good convention for now.

Reply:
Ack, let's document this then as well when you are at it eventually. @anvddriesch let's then suffix our test versions with our user names for now.

Reply:
That sounds like a good way to do it for me!

Reply:
Added note to the issue.